frame/transaction-storage/src/migration.rs

```diff
@@ -0,0 +1,47 @@
+use frame_support::traits::{Get, PalletInfoAccess, StorageVersion};
+
+use crate::*;
+
+pub struct RemoveLimitsFromStorage<T: crate::Config>(sp_std::marker::PhantomData<T>);
```

Comment:
Is this the preferred way for storage migrations? I was looking around and saw different methods used for different pallets 😬

Reply:
I can't answer you question, but I don't think migration is even needed here. There are still no deployed chains that use this crate as far as I know.

Reply:
yes, migration is not needed